### PR TITLE
volla-yggdrasil: update VollaOS

### PR DESCRIPTION
**Pull request by luka177**

### CODE REVIEW

File: v2/devices/yggdrasil.yml [hunk 1]
```diff
@@ -194,111 +194,91 @@ operating_systems:
...
+  - name: "Volla OS"
+    prerequisites: []
     steps:
       - actions:
           - core:download:
-              group: "VollaOS"
+              group: "firmware"
               files:
-                - url: "https://ota.volla.tech/volla-9.0-20201019-nightly-k63v2_64_bsp-signed.zip"
+                - url: "https://volla.tech/filedump/vollaos-10.0-ubports.zip"
```
Comment (NotKit) [inline v2/devices/yggdrasil.yml:204]: Perhaps you should add some versioning to filename in URL. This way we should be able to update it later without breaking checksum of existing config.

File: v2/devices/yggdrasil.yml [hunk 2]
```diff
@@ -194,111 +194,91 @@ operating_systems:
...
       - actions:
-          - core:user_action:
-              action: "twrp_sideload"
+          - fastboot:erase:
+              partition: "userdata"
       - actions:
-          - adb:sideload:
-              group: "VollaOS"
-              file: "volla-9.0-20201019-nightly-k63v2_64_bsp-signed.zip"
+          - fastboot:erase:
```
Comment (NotKit) [inline v2/devices/yggdrasil.yml:275]: erase, not format? Does Android format it to ext4 correctly on boot?

Comment (luka177) [reply v2/devices/yggdrasil.yml:275]: yes

File: v2/devices/yggdrasil.yml (hunk 1 above)
Comment (luka177) [reply v2/devices/yggdrasil.yml]: done

**Comment by Flohack74** [comment]
@luka177 @NotKit this is a major change, are we 100% sure it works?

**Comment by luka177** [comment]
yes it is tested

**Review by Flohack74 — APPROVED** [review]

**Pull request merged**